firmware/hw_layer/ports/kinetis/mpu_util.cpp

```diff
@@ -257,5 +257,10 @@ uintptr_t getFlashAddrSecondCopy() {
 	return 0x10008000;
 }
 
+/*static*/ hardware_pwm* hardware_pwm::tryInitPin(const char*, brain_pin_e, float, float) {
+	// TODO: implement me!
```

Comment:
what is the plan here?

Reply:
This isn't required to be implemented, it just means hardware PWM won't ever be used on Kinetis.